contributions/presentation/week6/agnespet-adahen/README.md

```diff
@@ -0,0 +1,16 @@
+
+# Presentation Proposal
```

Comment:
What is the novelty of your presentation? Can you explicitly make a novelty statement?

Reply:
I've updated the PR with better info